dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -55,6 +55,15 @@
 			};
 		};
 
+
+		uart: uart@50020000 {
+			compatible = "renesas,smartbond-uart";
+			reg = <0x50020000 0x100>;
+			periph-clock-config = <0x01>;
```

Comment:
for future: you should implement a proper clock control driver and use standard `clocks` property.